dojo/settings/settings.dist.py

```diff
@@ -175,7 +175,7 @@
 
     # Feature toggle for new authorization, which is incomplete at the moment.
     # Don't set it to True for productive environments!
-    DD_FEATURE_AUTHORIZATION_V2=(bool, False),
+    DD_FEATURE_AUTHORIZATION_V2=(bool, True),
```

Comment:
prefer to set this to True with the PR that adds some API testcases :-D

Reply:
I am currently working on API test cases, they will come this weekend. I would like to keep this PR here as it is and you merge it after the API tests. Otherwise the migration and the documentation would need to be changed as well.